tests/test_sklearn_sv_converter.py

```diff
@@ -126,6 +128,75 @@ def test_float64_linear_svc(self):
         self.assertTrue(torch_model is not None)
         np.testing.assert_allclose(model.predict(X), torch_model.predict(X), rtol=1e-6, atol=1e-6)
 
+    # Torchscript backend
+    def test_svc_ts(self):
+        self._test_svc(2, backend="torch.jit")
+
+    # SVC linear kernel
+    def test_svc_linear_ts(self):
+        self._test_svc(2, kernel="linear", backend="torch.jit")
+
+    # SVC sigmoid kernel
+    def test_svc_sigmoid_ts(self):
+        self._test_svc(2, kernel="sigmoid", backend="torch.jit")
+
+    # SVC poly kernel
+    def test_svc_poly_ts(self):
+        self._test_svc(2, kernel="poly", backend="torch.jit")
+
+    # NuSVC binary
+    def test_nu_svc_bi_ts(self):
+        self._test_nu_svc(2, backend="torch.jit")
+
+    def test_svc_multi_ts(self):
+        self._test_svc(3, backend="torch.jit")
+
+    # TVM backend.
+    # SVC binary
+    @unittest.skipIf(not (tvm_installed()), reason="TVM tests require TVM")
+    def test_svc_tvm(self):
+        self._test_svc(2, backend="tvm", extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+
+    # SVC linear kernel
+    @unittest.skipIf(not (tvm_installed()), reason="TVM tests require TVM")
+    def test_svc_linear_tvm(self):
+        self._test_svc(2, kernel="linear", backend="tvm", extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+
+    # SVC sigmoid kernel
+    @unittest.skipIf(not (tvm_installed()), reason="TVM tests require TVM")
+    def test_svc_sigmoid_tvm(self):
+        self._test_svc(2, kernel="sigmoid", backend="tvm", extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+
+    # SVC poly kernel
+    @unittest.skipIf(not (tvm_installed()), reason="TVM tests require TVM")
+    def test_svc_poly_tvm(self):
+        self._test_svc(2, kernel="poly", backend="tvm", extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+
+    # NuSVC binary
+    @unittest.skipIf(not (tvm_installed()), reason="TVM tests require TVM")
+    def test_nu_svc_bi_tvm(self):
+        self._test_nu_svc(2, backend="tvm", extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+
+    # # SVC multiclass
```

Comment:
future works?

Reply:
Yes, we are waiting for [some](https://github.com/pytorch/pytorch/issues/43867) operator in PyTorch.

Reply:
ah ok cool!  i know you have a comment somewhere else, but can you please link to that  here too?

Reply:
yep. Done.